proto/ibc/connection/connection.proto

```diff
@@ -39,12 +39,14 @@ message MsgConnectionOpenTry {
   // proof of the initialization the connection on Chain A: `UNITIALIZED ->
   // INIT`
   bytes proof_init = 5 [(gogoproto.moretags) = "yaml:\"proof_init\""];
-  uint64 proof_height = 6;
+  uint64 proof_epoch  = 6 [(gogoproto.moretags) = "yaml:\"proof_epoch\""];
+  uint64 proof_height = 7;
   // proof of client consensus state
```

Comment:
The Epoch and height are passed in seperately in these proto msgs, and then passed into `clientexported.Height` constructor in code